task/python-sdk-azure/0.1/README.md

```diff
@@ -0,0 +1,86 @@
+**Tekton Task for Running Python Scripts with Azure SDK**
+
+This Tekton Task enables execution of Python scripts that utilize the Azure SDK to interact with Azure services. The Task is designed to securely authenticate using Azure credentials and execute the script from a ConfigMap without embedding the script directly in the pipeline.
+
+
+**Features**
```

Comment:
hi @arunvel1988, while having the README in the current state is fine, will it be possible for you to use proper markdown headings? You can probably refer a sample readme - https://raw.githubusercontent.com/tektoncd/catalog/refs/heads/main/task/anchore-cli/0.1/README.md

Reply:
@vinamra28 done